.github/ISSUE_TEMPLATE/new-repo-checklist.md

```diff
@@ -62,10 +64,10 @@ https://github.com/maplibre/[repo]
   *Use the gear icon in the upper right corner of the repo page.*
 - [ ] CI automatically runs on all pull requests before merging using GitHub actions
 - [ ] Grant admin rights to the board members and automation accounts for packages <list-of-packages>
-  - [npmjs.com](https://www.npmjs.com/): package settings / invite:  `maplibreorg nyurik klokan lseelenbinder wipfli`
-  - [crates.io](https://crates.io/): package settings / add owner: `nyurik klokan lseelenbinder wipfli`
+    - [npmjs.com](https://www.npmjs.com/): package settings / invite:  `maplibreorg nyurik klokan lseelenbinder wipfli`
+    - [crates.io](https://crates.io/): package settings / add owner: `nyurik klokan lseelenbinder wipfli`
 
 ## Community
 - [ ] The new repo has been announced in the `#maplibre` OSMUS slack channel.
 - [ ] The new repo has been announced in the next monthly meeting of the Technical Steering Committee.
-- [ ] The new repo has been announced in the `@maplibre` twitter.
+- [ ] The new repo has been announced in the [@maplibre.org](https://bsky.app/profile/maplibre.org) Bluesky.
```

Comment:
```suggestion
- [ ] The new repo has been announced on our social networks.
```

I would add an item: has been announced in the newsletter.

I would just reformulate it as "has been announced on social media". We also use X, Mastodon and LinkedIn.